src/test/regress/expected/multi_insert_select.out

```diff
@@ -1228,39 +1230,46 @@ FROM   raw_events_first;
 DEBUG:  StartTransactionCommand
 DEBUG:  StartTransaction
 DEBUG:  name: unnamed; blockState:       DEFAULT; state: INPROGR, xid/subid/cid: 0/1/0, nestlvl: 1, children: 
-ERROR:  SELECT query should return bare partition column on the same ordinal position as the INSERT's partition column
+ERROR:  cannot plan distributed INSERT INTO ... SELECT query because of partition column restrictions
```

Comment:
We had a session with @samay-sharma, discussed the potential improvements. Most probably, we'll sit together again finalize the wording today.